Makefile

```diff
@@ -18,17 +19,25 @@ linklist.o:
 	$(CC) $(CCFLAGS) -c $(LIST)/linklist.c \
 	-o $(OBJFILE)/linklist.o -I $(INCLUDE)
 
-main: string.o linklist.o
+sorting.o:
+	$(CC) $(CCFLAGS) -c $(SORT)/sorting.c \
+	-o $(OBJFILE)/sorting.o -I $(INCLUDE)
+
+main: string.o linklist.o sorting.o
 	$(CC) $(CCFLAGS) $(STRING)/main.c $(OBJFILE)/string.o \
 	-o $(BIN)/stringMain -I $(INCLUDE)
 	$(CC) $(CCFLAGS) $(LIST)/main.c $(OBJFILE)/linklist.o \
 	-o $(BIN)/linklistMain -I $(INCLUDE)
+	$(CC) $(CCFLAGS) $(SORT)/main.c $(OBJFILE)/sorting.o \
+	-o $(BIN)/sortingMain -I $(INCLUDE)
 
 run:
-	@echo -e "Running string programs ...\n"
+	@echo -e "\nRunning string programs ...\n"
 	$(BIN)/stringMain
-	@echo -e "Running linklist programs ...\n"
+	@echo -e "\nRunning linklist programs ...\n"
 	$(BIN)/linklistMain
+	@echo -e "\nRunning sorting programs ...\n"
+	$(BIN)/sortingMain
```

Comment:
make prints -e, while echoing statements. Please check why? 
I must have introduced this bug by mistake